healthcare-management/src/app/app-config/app.config.ts

```diff
@@ -1,6 +1,7 @@
-import {environment} from '../../environments/environment';
+
 import {AppConfig} from './app.config.interface';
 import {InjectionToken} from '@angular/core';
+import {environment} from '../../environments/environment.development';
```

Comment:
Am uitat de mentionat, aici va trebui lasat import {environment} from '../../environments/environment'; deoarece acum vom avea doar un singur environment (poti sa redenumesti environment.development in environment). Am facut asta pentru deploy-ul pe vercel